nextjs/scripts/create-local-server.js

```diff
@@ -45,8 +44,10 @@ app
         logger: console,
         target: pubHost,
         changeOrigin: true,
-        secure: !isDevelopment,
+        secure: !dev,
         onProxyReq: (proxyReq, request, response) => {
+          const ip = (request.headers['x-forwarded-for'] || request.socket.remoteAddress).split(':').pop()
+          proxyReq.setHeader('originalIP', ip );
```

Comment:
This is so the Next app will tell the rails app the original IP for admin verification.